diesel/src/sqlite/expression/expression_methods.rs

```diff
@@ -43,8 +45,10 @@ pub trait SqliteExpressionMethods: Expression + Sized {
 
     /// Creates a Sqlite `IS NOT` expression.
     ///
-    /// This behaves identically to the `!=` operator, except that `NULL` is
-    /// treated as a normal value.
+    /// The `IS NOT` operator work like != except when one or both of the operands are NULL.
+    /// In this case, if both operands are NULL, then the `IS NOT` operator evaluates to 0 (false).
+    /// If one operand is NULL and the other is not, then the `IS NOT` operator is 1 (true).
```

Comment:
As diesel handles the `0`-> `false`, `1` -> `true` mapping here I would prefer to just use the boolean values here:
```suggestion
    /// In this case, if both operands are NULL, then the `IS NOT` operator evaluates to false.
    /// If one operand is NULL and the other is not, then the `IS NOT` operator is true.
```